From a46726f43123659a2c3ace01d52bf9a53b6aaa76 Mon Sep 17 00:00:00 2001
From: Ingo Schwarze <schwarze@openbsd.org>
Date: Sat, 21 Dec 2019 14:39:17 +0100
Subject: Update NetBSD, OpenBSD, FreeBSD, Darwin, and DragonFly version
 strings.

* tmac/doc-common-u: Update.
* tmac/groff_mdoc.7.man: Synchronize.

Based on a patch from Guillem Jover <guillem@debian.org>
via Colin Watson <cjwatson@debian.org>, tweaked by me.
Fixes: https://bugs.debian.org/867123

Origin: upstream, https://git.savannah.gnu.org/cgit/groff.git/commit/?id=5a8af7104f1c581bcfbad12b56033ad403b0afe1
Bug-Debian: https://bugs.debian.org/867123
Last-Update: 2019-12-23

Patch-Name: bsd-updates.patch
---
 tmac/doc-common-u     | 122 ++++++++++++++++++++++++++++++++++++++++++
 tmac/groff_mdoc.7.man |  38 ++++++++-----
 2 files changed, 147 insertions(+), 13 deletions(-)

diff --git a/tmac/doc-common-u b/tmac/doc-common-u
index 0d2e418d..648926c6 100644
--- a/tmac/doc-common-u
+++ b/tmac/doc-common-u
@@ -499,11 +499,22 @@
 .ds doc-operating-system-NetBSD-6.0.3 6.0.3
 .ds doc-operating-system-NetBSD-6.0.4 6.0.4
 .ds doc-operating-system-NetBSD-6.0.5 6.0.5
+.ds doc-operating-system-NetBSD-6.0.6 6.0.6
 .ds doc-operating-system-NetBSD-6.1   6.1
 .ds doc-operating-system-NetBSD-6.1.1 6.1.1
 .ds doc-operating-system-NetBSD-6.1.2 6.1.2
 .ds doc-operating-system-NetBSD-6.1.3 6.1.3
 .ds doc-operating-system-NetBSD-6.1.4 6.1.4
+.ds doc-operating-system-NetBSD-6.1.5 6.1.5
+.ds doc-operating-system-NetBSD-7.0   7.0
+.ds doc-operating-system-NetBSD-7.0.1 7.0.1
+.ds doc-operating-system-NetBSD-7.0.2 7.0.2
+.ds doc-operating-system-NetBSD-7.1   7.1
+.ds doc-operating-system-NetBSD-7.1.1 7.1.1
+.ds doc-operating-system-NetBSD-7.1.2 7.1.2
+.ds doc-operating-system-NetBSD-7.2   7.2
+.ds doc-operating-system-NetBSD-8.0   8.0
+.ds doc-operating-system-NetBSD-8.1   8.1
 .
 .ds doc-operating-system-OpenBSD-2.0  2.0
 .ds doc-operating-system-OpenBSD-2.1  2.1
@@ -542,6 +553,16 @@
 .ds doc-operating-system-OpenBSD-5.4  5.4
 .ds doc-operating-system-OpenBSD-5.5  5.5
 .ds doc-operating-system-OpenBSD-5.6  5.6
+.ds doc-operating-system-OpenBSD-5.7  5.7
+.ds doc-operating-system-OpenBSD-5.8  5.8
+.ds doc-operating-system-OpenBSD-5.9  5.9
+.ds doc-operating-system-OpenBSD-6.0  6.0
+.ds doc-operating-system-OpenBSD-6.1  6.1
+.ds doc-operating-system-OpenBSD-6.2  6.2
+.ds doc-operating-system-OpenBSD-6.3  6.3
+.ds doc-operating-system-OpenBSD-6.4  6.4
+.ds doc-operating-system-OpenBSD-6.5  6.5
+.ds doc-operating-system-OpenBSD-6.6  6.6
 .
 .ds doc-operating-system-FreeBSD-1.0     1.0
 .ds doc-operating-system-FreeBSD-1.1     1.1
@@ -608,6 +629,16 @@
 .ds doc-operating-system-FreeBSD-9.2     9.2
 .ds doc-operating-system-FreeBSD-9.3     9.3
 .ds doc-operating-system-FreeBSD-10.0    10.0
+.ds doc-operating-system-FreeBSD-10.1    10.1
+.ds doc-operating-system-FreeBSD-10.2    10.2
+.ds doc-operating-system-FreeBSD-10.3    10.3
+.ds doc-operating-system-FreeBSD-10.4    10.4
+.ds doc-operating-system-FreeBSD-11.0    11.0
+.ds doc-operating-system-FreeBSD-11.1    11.1
+.ds doc-operating-system-FreeBSD-11.2    11.2
+.ds doc-operating-system-FreeBSD-11.3    11.3
+.ds doc-operating-system-FreeBSD-12.0    12.0
+.ds doc-operating-system-FreeBSD-12.1    12.1
 .
 .ds doc-operating-system-Darwin-8.0.0  8.0.0
 .ds doc-operating-system-Darwin-8.1.0  8.1.0
@@ -654,6 +685,44 @@
 .ds doc-operating-system-Darwin-13.3.0 13.3.0
 .ds doc-operating-system-Darwin-13.4.0 13.4.0
 .ds doc-operating-system-Darwin-14.0.0 14.0.0
+.ds doc-operating-system-Darwin-14.1.0 14.1.0
+.ds doc-operating-system-Darwin-14.2.0 14.2.0
+.ds doc-operating-system-Darwin-14.3.0 14.3.0
+.ds doc-operating-system-Darwin-14.4.0 14.4.0
+.ds doc-operating-system-Darwin-14.5.0 14.5.0
+.ds doc-operating-system-Darwin-15.0.0 15.0.0
+.ds doc-operating-system-Darwin-15.1.0 15.1.0
+.ds doc-operating-system-Darwin-15.2.0 15.2.0
+.ds doc-operating-system-Darwin-15.3.0 15.3.0
+.ds doc-operating-system-Darwin-15.4.0 15.4.0
+.ds doc-operating-system-Darwin-15.5.0 15.5.0
+.ds doc-operating-system-Darwin-15.6.0 15.6.0
+.ds doc-operating-system-Darwin-16.0.0 16.0.0
+.ds doc-operating-system-Darwin-16.1.0 16.1.0
+.ds doc-operating-system-Darwin-16.2.0 16.2.0
+.ds doc-operating-system-Darwin-16.3.0 16.3.0
+.ds doc-operating-system-Darwin-16.4.0 16.4.0
+.ds doc-operating-system-Darwin-16.5.0 16.5.0
+.ds doc-operating-system-Darwin-16.6.0 16.6.0
+.ds doc-operating-system-Darwin-17.0.0 17.0.0
+.ds doc-operating-system-Darwin-17.1.0 17.1.0
+.ds doc-operating-system-Darwin-17.2.0 17.2.0
+.ds doc-operating-system-Darwin-17.3.0 17.3.0
+.ds doc-operating-system-Darwin-17.4.0 17.4.0
+.ds doc-operating-system-Darwin-17.5.0 17.5.0
+.ds doc-operating-system-Darwin-17.6.0 17.6.0
+.ds doc-operating-system-Darwin-17.7.0 17.7.0
+.ds doc-operating-system-Darwin-18.0.0 18.0.0
+.ds doc-operating-system-Darwin-18.1.0 18.1.0
+.ds doc-operating-system-Darwin-18.2.0 18.2.0
+.ds doc-operating-system-Darwin-18.3.0 18.3.0
+.ds doc-operating-system-Darwin-18.4.0 18.4.0
+.ds doc-operating-system-Darwin-18.5.0 18.5.0
+.ds doc-operating-system-Darwin-18.6.0 18.6.0
+.ds doc-operating-system-Darwin-18.7.0 18.7.0
+.ds doc-operating-system-Darwin-19.0.0 19.0.0
+.ds doc-operating-system-Darwin-19.1.0 19.1.0
+.ds doc-operating-system-Darwin-19.2.0 19.2.0
 .
 .ds doc-operating-system-DragonFly-1.0    1.0
 .ds doc-operating-system-DragonFly-1.1    1.1
@@ -688,14 +757,67 @@
 .ds doc-operating-system-DragonFly-2.12   2.12
 .ds doc-operating-system-DragonFly-2.13   2.13
 .ds doc-operating-system-DragonFly-3.0    3.0
+.ds doc-operating-system-DragonFly-3.0.1  3.0.1
+.ds doc-operating-system-DragonFly-3.0.2  3.0.2
 .ds doc-operating-system-DragonFly-3.1    3.1
 .ds doc-operating-system-DragonFly-3.2    3.2
+.ds doc-operating-system-DragonFly-3.2.1  3.2.1
+.ds doc-operating-system-DragonFly-3.2.2  3.2.2
 .ds doc-operating-system-DragonFly-3.3    3.3
 .ds doc-operating-system-DragonFly-3.4    3.4
+.ds doc-operating-system-DragonFly-3.4.1  3.4.1
+.ds doc-operating-system-DragonFly-3.4.2  3.4.2
+.ds doc-operating-system-DragonFly-3.4.3  3.4.3
 .ds doc-operating-system-DragonFly-3.5    3.5
 .ds doc-operating-system-DragonFly-3.6    3.6
+.ds doc-operating-system-DragonFly-3.6.1  3.6.1
+.ds doc-operating-system-DragonFly-3.6.2  3.6.2
 .ds doc-operating-system-DragonFly-3.7    3.7
 .ds doc-operating-system-DragonFly-3.8    3.8
+.ds doc-operating-system-DragonFly-3.8.1  3.8.1
+.ds doc-operating-system-DragonFly-3.8.2  3.8.2
+.ds doc-operating-system-DragonFly-4.0    4.0
+.ds doc-operating-system-DragonFly-4.0.1  4.0.1
+.ds doc-operating-system-DragonFly-4.0.2  4.0.2
+.ds doc-operating-system-DragonFly-4.0.3  4.0.3
+.ds doc-operating-system-DragonFly-4.0.4  4.0.4
+.ds doc-operating-system-DragonFly-4.0.5  4.0.5
+.ds doc-operating-system-DragonFly-4.0.6  4.0.6
+.ds doc-operating-system-DragonFly-4.1    4.1
+.ds doc-operating-system-DragonFly-4.2    4.2
+.ds doc-operating-system-DragonFly-4.2.1  4.2.1
+.ds doc-operating-system-DragonFly-4.2.2  4.2.2
+.ds doc-operating-system-DragonFly-4.2.3  4.2.3
+.ds doc-operating-system-DragonFly-4.2.4  4.2.4
+.ds doc-operating-system-DragonFly-4.3    4.3
+.ds doc-operating-system-DragonFly-4.4    4.4
+.ds doc-operating-system-DragonFly-4.4.1  4.4.1
+.ds doc-operating-system-DragonFly-4.4.2  4.4.2
+.ds doc-operating-system-DragonFly-4.4.3  4.4.3
+.ds doc-operating-system-DragonFly-4.5    4.5
+.ds doc-operating-system-DragonFly-4.6    4.6
+.ds doc-operating-system-DragonFly-4.6.1  4.6.1
+.ds doc-operating-system-DragonFly-4.6.2  4.6.2
+.ds doc-operating-system-DragonFly-4.7    4.7
+.ds doc-operating-system-DragonFly-4.8    4.8
+.ds doc-operating-system-DragonFly-4.8.1  4.8.1
+.ds doc-operating-system-DragonFly-4.9    4.9
+.ds doc-operating-system-DragonFly-5.0    5.0
+.ds doc-operating-system-DragonFly-5.0.1  5.0.1
+.ds doc-operating-system-DragonFly-5.0.2  5.0.2
+.ds doc-operating-system-DragonFly-5.1    5.1
+.ds doc-operating-system-DragonFly-5.2    5.2
+.ds doc-operating-system-DragonFly-5.2.1  5.2.1
+.ds doc-operating-system-DragonFly-5.2.2  5.2.2
+.ds doc-operating-system-DragonFly-5.3    5.3
+.ds doc-operating-system-DragonFly-5.4    5.4
+.ds doc-operating-system-DragonFly-5.4.1  5.4.1
+.ds doc-operating-system-DragonFly-5.4.2  5.4.2
+.ds doc-operating-system-DragonFly-5.4.3  5.4.3
+.ds doc-operating-system-DragonFly-5.5    5.5
+.ds doc-operating-system-DragonFly-5.6    5.6
+.ds doc-operating-system-DragonFly-5.6.1  5.6.1
+.ds doc-operating-system-DragonFly-5.6.2  5.6.2
 .
 .de Os
 .  ds doc-command-name
diff --git a/tmac/groff_mdoc.7.man b/tmac/groff_mdoc.7.man
index 2e9824dc..05f895f9 100644
--- a/tmac/groff_mdoc.7.man
+++ b/tmac/groff_mdoc.7.man
@@ -879,28 +879,40 @@ the release ID.
 1.3, 1.3a, 1.4, 1.4.1, 1.4.2, 1.4.3, 1.5, 1.5.1, 1.5.2, 1.5.3, 1.6, 1.6.1,
 1.6.2, 1.6.3, 2.0, 2.0.1, 2.0.2, 2.0.3, 2.1, 3.0, 3.0.1, 3.0.2, 3.0.3,
 3.1, 3.1.1, 4.0, 4.0.1, 5.0, 5.0.1, 5.0.2, 5.1, 5.1.2, 5.1.3, 5.1.4,
-5.2, 5.2.1, 5.2.2, 6.0, 6.0.1, 6.0.2, 6.0.3, 6.0.4, 6.0.5, 6.1, 6.1.1,
-6.1.2, 6.1.3, 6.1.4
+5.2, 5.2.1, 5.2.2, 6.0, 6.0.1, 6.0.2, 6.0.3, 6.0.4, 6.0.5, 6.0.6, 6.1,
+6.1.1, 6.1.2, 6.1.3, 6.1.4, 6.1.5, 7.0, 7.0.1, 7.0.2, 7.1, 7.1.1, 7.1.2,
+7.2, 8.0, 8.1
 .It FreeBSD
 1.0, 1.1, 1.1.5, 1.1.5.1, 2.0, 2.0.5, 2.1, 2.1.5, 2.1.6, 2.1.7, 2.2, 2.2.1,
-2.2.2, 2.2.5, 2.2.6, 2.2.7, 2.2.8, 3.0, 3.1, 3.2, 3.3, 3.4, 3.5, 4.0, 4.1,
-4.1.1, 4.2, 4.3, 4.4, 4.5, 4.6, 4.6.2, 4.7, 4.8, 4.9, 4.10, 4.11, 5.0, 5.1,
-5.2, 5.2.1, 5.3, 5.4, 5.5, 6.0, 6.1, 6.2, 6.3, 6.4, 7.0, 7.1, 7.2, 7.3, 7.4,
-8.0, 8.1, 8.2, 8.3, 8.4, 9.0, 9.1, 9.2, 9.3, 10.0
+2.2.2, 2.2.5, 2.2.6, 2.2.7, 2.2.8, 2.2.9, 3.0, 3.1, 3.2, 3.3, 3.4, 3.5, 4.0,
+4.1, 4.1.1, 4.2, 4.3, 4.4, 4.5, 4.6, 4.6.2, 4.7, 4.8, 4.9, 4.10, 4.11, 5.0,
+5.1, 5.2, 5.2.1, 5.3, 5.4, 5.5, 6.0, 6.1, 6.2, 6.3, 6.4, 7.0, 7.1, 7.2, 7.3,
+7.4, 8.0, 8.1, 8.2, 8.3, 8.4, 9.0, 9.1, 9.2, 9.3, 10.0, 10.1, 10.2, 10.3,
+10.4, 11.0, 11.1, 11.2, 11.3, 12.0, 12.1
 .It OpenBSD
 2.0, 2.1, 2.2, 2.3, 2.4, 2.5, 2.6, 2.7, 2.8, 2.9, 3.0, 3.1, 3.2, 3.3, 3.4,
 3.5, 3.6, 3.7, 3.8, 3.9, 4.0, 4.1, 4.2, 4.3, 4.4, 4.5, 4.6, 4.7, 4.8, 4.9,
-5.0, 5.1, 5.2, 5.3, 5.4, 5.5, 5.6
+5.0, 5.1, 5.2, 5.3, 5.4, 5.5, 5.6, 5.7, 5.8, 5.9, 6.0, 6.1, 6.2, 6.3, 6.4,
+6.5, 6.6
 .It DragonFly
-1.0, 1.1, 1.2, 1.3, 1.4, 1.5, 1.6, 1.8, 1.8.1, 1.9, 1.10, 1.12, 1.12.2,
-1.13, 2.0, 2.2, 2.3, 2.4, 2.5, 2.6, 2.7, 2.8, 2.9, 2.9.1, 2.10, 2.10.1,
-2.11, 2.12, 2.13, 3.0, 3.1, 3.2, 3.3, 3.4, 3.5, 3.6, 3.7, 3.8
+1.0, 1.1, 1.2, 1.3, 1.4, 1.5, 1.6, 1.7, 1.8, 1.8.1, 1.9, 1.10, 1.11, 1.12,
+1.12.2, 1.13, 2.0, 2.1, 2.2, 2.3, 2.4, 2.5, 2.6, 2.7, 2.8, 2.9, 2.9.1, 2.10,
+2.10.1, 2.11, 2.12, 2.13, 3.0, 3.0.1, 3.0.2, 3.1, 3.2, 3.2.1, 3.2.2, 3.3,
+3.4, 3.4.1, 3.4.2, 3.4.3, 3.5, 3.6, 3.6.1, 3.6.2, 3.7, 3.8, 3.8.1, 3.8.2,
+4.0, 4.0.1, 4.0.2, 4.0.3, 4.0.4, 4.0.5, 4.0.6, 4.1, 4.2, 4.2.1, 4.2.2,
+4.2.3, 4.2.4, 4.3, 4.4, 4.4.1, 4.4.2, 4.4.3, 4.5, 4.6, 4.6.1, 4.6.2, 4.7,
+4.8, 4.8.1, 4.9, 5.0, 5.0.1, 5.0.2, 5.1, 5.2, 5.2.1, 5.2.2, 5.3, 5.4, 5.4.1,
+5.4.2, 5.4.3, 5.5, 5.6, 5.6.1, 5.6.2
 .It Darwin
 8.0.0, 8.1.0, 8.2.0, 8.3.0, 8.4.0, 8.5.0, 8.6.0, 8.7.0, 8.8.0, 8.9.0,
 8.10.0, 8.11.0, 9.0.0, 9.1.0, 9.2.0, 9.3.0, 9.4.0, 9.5.0, 9.6.0, 9.7.0,
-9.8.0, 10.1.0, 10.2.0, 10.3.0, 10.4.0, 10.5.0, 10.6.0, 10.7.0, 10.8.0,
-11.0.0, 11.1.0, 11.2.0, 11.3.0, 11.4.0, 11.5.0, 12.0.0, 12.1.0, 12.2.0,
-13.0.0, 13.1.0, 13.2.0, 13.3.0, 13.4.0, 14.0.0
+9.8.0, 10.0.0, 10.1.0, 10.2.0, 10.3.0, 10.4.0, 10.5.0, 10.6.0, 10.7.0,
+10.8.0, 11.0.0, 11.1.0, 11.2.0, 11.3.0, 11.4.0, 11.5.0, 12.0.0, 12.1.0,
+12.2.0, 13.0.0, 13.1.0, 13.2.0, 13.3.0, 13.4.0, 14.0.0, 14.1.0, 14.2.0,
+14.3.0, 14.4.0, 14.5.0, 15.0.0, 15.1.0, 15.2.0, 15.3.0, 15.4.0, 15.5.0,
+15.6.0, 16.0.0, 16.1.0, 16.2.0, 16.3.0, 16.4.0, 16.5.0, 16.6.0, 17.0.0,
+17.1.0, 17.2.0, 17.3.0, 17.4.0, 17.5.0, 17.6.0, 17.7.0, 18.0.0, 18.1.0,
+18.2.0, 18.3.0, 18.4.0, 18.5.0, 18.6.0, 18.7.0, 19.0.0, 19.1.0, 19.2.0
 .El
 .Ed
 .Pp
